JumpscaleCore/core/BASECLASSES/JSConfigBCDB.py

```diff
@@ -109,11 +108,8 @@ def _delete(self, recursive=None):
                 if not isinstance(self._parent, j.baseclasses.factory):
                     # if factory then cannot delete from the mother because its the only object
                     del self._parent._children[self._data.name]
-        mother_id = self._mother_id_get()
```

Comment:
I don't get this logic. If I am deleting an object, then I should delete its children. The fact that an object has a mother doesn't relate to having to delete the object's children.